Updates to download guide: adding dedicated upload section (#7328)
DataDog/documentation: content/en/synthetics/guide/testing-file-upload-and-download.md

      text: 'Configure a Browser Test'
---

> Web applications can embed a lot of logic, and although end-to-end tests are often mostly made of basic interactions (eg. clicks, input forms, ...) with your website, you sometimes need to go one step further and verify complex functionalities in order to ensure key business transactions can be performed on your application.

Reviewer: ```suggestion
Web applications can embed a lot of logic, and although end-to-end tests are often mostly made of basic interactions (for example, clicks and input forms) for testing your website, you sometimes need to go one step further and verify complex interactions to ensure key business transactions can be performed on your application.
```

## Testing a file upload

> You may, for instance, need to **upload a file** to validate the final step of a functional workflow to test a profile creation. When uploading a file at the test recorder level, Datadog Synthetics Browser tests automatically identify the uploaded file and create the [`Upload file` associated step][1]. It is then able to upload that file again at test execution.

Reviewer: ```suggestion
You can **upload a file** to validate the final step of a functional workflow to test a profile creation. When uploading a file at the test recorder level, Datadog Synthetics Browser tests automatically identify the uploaded file and create the [`Upload file` associated step][1]. It is then able to upload that file again at test execution.
```
